### ISSUE TITLE
artifactory_permission_target does not support release_bundles

### ISSUE THREAD

**Issue by neilvana** [jfrog/terraform-provider-artifactory#449]
**Is your feature request related to a problem? Please describe.**
I cannot use terraform to configure permission targets which manage release bundles because the terraform provider does not support it.  To make matters more difficult, the schema implemented within the provider does support release_bundles but this configuration is silently ignored when applying the resource state causing inconsistency between the state and the resource configuration.

**Describe the solution you'd like**
I would like to be able to configure permission targets which manage permissions for release bundles.  I plan to put up a PR shortly which adds this support, so really I would like a review and collaboration to get it integrated.

**Describe alternatives you've considered**
I need to manually update the target either using  the Rest API or the UI.

**Additional context**
I plan to put up a PR to the provider when I have time to implement it, but I wanted to get the issue filed so I could document it in my terraform code in the interim.

**Comment by chb0github** [comment]
Do you have some HCL you'd like to propose?

We welcome your PR

**Comment by chb0github** [comment]
Our general position is: Anything we can validate/verify client side, we do. So, in the case where only certain values are allowed based on other selections, this has sometimes meant that we have created entirely different resources that can have wholly different schema (like the way we manage repos)

@alexhung - do you think this applies here?

**Comment by alexhung** [comment]
@chb0github No, because the schema for `repo`, `build`, and `releaseBundle` are identical (see the [V2 schema](https://www.jfrog.com/confluence/display/JFROG/Security+Configuration+JSON#SecurityConfigurationJSON-application/vnd.org.jfrog.artifactory.security.PermissionTargetV2+json)) and the current validations are adequate.
